### PR TITLE
chore(js): implement a structure & blueprint for javascript modules

### PR DESCRIPTION
**Pull request by Wandkleister**

### CODE REVIEW

File: _includes/default.liquid [hunk 1]
```diff
@@ -4,6 +4,7 @@
 	<body>
 		<main>
 			{{ content }}
+			{% include slider/slider %}
```
Comment (luismeyer) [inline _includes/default.liquid:7]: Shouldn't the different sections be included from the index page? I'm not sure if i understood the onboarding wrong but i thought the default.liquid is more like a general template for pages and the index page is the actual page. Correct me if i'm wrong 😄

Comment (Wandkleister) [reply _includes/default.liquid]: Yes you're completely right! Including it now from the index.liquid page. Thank you :)

File: package.json [hunk 2]
```diff
@@ -5,9 +5,11 @@
   "description": "SinnerSchrader website reboot",
   "main": "index.js",
   "scripts": {
-    "build": "npx @11ty/eleventy",
-    "watch": "npx @11ty/eleventy --serve",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "build": "npx @11ty/eleventy && npm run bundlejs:prod",
+    "watch": "npx @11ty/eleventy --serve && npm run bundlejs:dev",
```
Comment (timbru31) [inline package.json:9]: Does that work? The watch task will never finish until cancelled, i.e., the second command is not run

Comment (Wandkleister) [reply package.json]: Reworked that. I added another script "start:dev" which runs the watch and the bundlejs:dev script

File: _includes/head.liquid [hunk 3]
```diff
@@ -1,5 +1,6 @@
 <head>
 	<link rel="stylesheet" type="text/css" href="{{ './index.css' | url }}" />
 	<meta name="viewport" content="width=device-width, initial-scale=1">
-	<meta charset="utf-8" />
+  <meta charset="utf-8" />
```
Comment (timbru31) [inline _includes/head.liquid:4]: other than this formatting change all good

```suggestion
	<meta charset="utf-8" />
```

Comment (Wandkleister) [reply _includes/head.liquid]: Fixed

**Pull request merged**